functional_tests/steps/release_page.py

```diff
@@ -116,6 +116,17 @@ def check_selected_datasets_are_displayed(context: Context):
         expect(context.page.get_by_text(dataset["description"])).to_be_visible()
 
 
+@when('the page status is set to "Provisional" and the release date text field is visible')
```

Comment:
nitpick: perhaps make this a `@step`?

My understanding of BDD is:
- `given` adds context. This is where you can either preset things (create user, page, etc) and do some actions (e.g. log in)
- `when` sets the scene. That is does actions
- `then` is the bit that checks the results.


I wonder if perhaps this scenario needs to be two:
1. when status is provisional, release date text is visiblwe
2. when changing to any other, it gets hidden

Reply:
Other than this, LGTM!

Reply:
I was trying to be overly concise here - I agree it makes sense to have it as two scenarios. 
I'm not too sure about making it a `@step` as 1 - it's only used once (with `Then`), and 2 - it only holds assertions so according to what you said here it's most likely going to stay a `Then`.

Reply:
[2c0d58a](https://github.com/ONSdigital/dis-wagtail/pull/158/commits/2c0d58acd74d8c6d51a23b1639ad34ac4046e402)